ospec/ospec.js

```diff
@@ -67,7 +67,7 @@ else window.o = m()
 		}
 		if (ospecFileName == null) return stack.join("\n")
 		// skip ospec-related entries on the stack
-		while (stack[i].indexOf(ospecFileName) !== -1) i++
+		while (stack[i+1] && stack[i].indexOf(ospecFileName) !== -1) i++
```

Comment:
Assuming we get internal errors again in the future (even if it is highly unlikely if we handle timeout properly), it would be better to let it run up to `stack.length` and return `error.stack` if `i === stack.length`.